### ISSUE TITLE
Updates about SOP for leaving/reapply of Reps

### ISSUE THREAD

**Issue by Mte90** [mozilla/Reps#309]
## Goal: 
During a call we discussed a lot about https://wiki.mozilla.org/ReMo/SOPs/Leaving as example that need to be updated also from our aspectatives (we have different docs working on that will require an update on this SOP).

Also from the onboarding side there are different thougts like for the pending references.

## Related Issues & Links: 
[Reference meta issues and past issues] #

## Roles:

Responsible:

Accountable:

Supporting:

Consulted:

Informed: 

## Required: 
- [x] Publish the new sop
- [x] Publish on Discourse
- [x] Publish on mailing list

**Comment by Mte90** [comment]
We have a case of a Reps that is alumni where the Council suggested to do a new  application to be onboarded again. In that case we are talking of a rep that is not active since more then 6 months (https://wiki.mozilla.org/ReMo/Application_Process/Selection_Criteria).

In the past year we had cases of alumni that want to come back and looking on their applications we used different procedure based on the period.
In one case there was the action of the onboarding team and in the other one no.
Also in the cases with the screening of the onboarding team we managed like a new application in all the steps so when he wasn't replying after 3 weeks we closed the application and moved back to alumni.

We need to update the SOP about leaving and reonboarding (https://wiki.mozilla.org/ReMo/SOPs/Leaving) to avoid cases like them and be sure that the alumni rep is active also after the moving to the rep group.
